### YamlMime:ManagedReference
items:
- uid: System.Workflow.Runtime.WorkflowOwnershipException
  id: WorkflowOwnershipException
  children:
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String,System.Exception)
  - System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String,System.Exception)
  - System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  langs:
  - csharp
  name: WorkflowOwnershipException
  nameWithType: WorkflowOwnershipException
  fullName: System.Workflow.Runtime.WorkflowOwnershipException
  type: Class
  summary: "Unntaket oppstod når arbeidsflyten runtime databasemotoren prøver å laste inn en arbeidsflytforekomst som er lastet inn av en annen runtime motoren arbeidsflytforekomsten. I tillegg unntak denne under kjøring arbeidsflytmotoren forsøk på å lagre en arbeidsflyt etter eierskap tidsavbruddet som ble angitt under innlasting av arbeidsflyten er utløpt."
  remarks: "> [!NOTE] >  [!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]WorkflowOwnershipException brukes av utholdenhet tjenester som støtter gjør mulighet for flere programmer for å få tilgang til samme database. I slike et miljø kan en forekomst av arbeidsflyten kjøretiden prøver å laste inn en arbeidsflyt som allerede er lastet inn av en annen forekomst av arbeidsflyten runtime motoren. For eksempel den <xref:System.Workflow.Runtime.Hosting.SQLWorkflowPersistenceService>klassen oppstår WorkflowOwnershipException når dette skjer.</xref:System.Workflow.Runtime.Hosting.SQLWorkflowPersistenceService>       Hvis du implementerer en utholdenhet-tjeneste som oppstår i WorkflowOwnershipException du må oppgi den <xref:System.Guid>til arbeidsflytforekomsten som forekommer unntaket ved hjelp av en riktig konstruktør for klassen eller innstillingen det <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>egenskapen før du throw unntaket.</xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Guid>"
  syntax:
    content: >-
      [System.Obsolete("The System.Workflow.* types are deprecated.  Instead, please use the new types from System.Activities.*")]

      public class WorkflowOwnershipException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor
  id: '#ctor'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException()
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException()
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException()
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> klasse."
  remarks: "Denne konstruktøren initialiserer den <xref:System.Exception.Message%2A>for den nye forekomsten til en system-angitt melding som beskriver problemet og tar hensyn til den gjeldende kulturen i systemet.</xref:System.Exception.Message%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Et nullreferanse (`Nothing` i Visual Basic) |   | <xref:System.Exception.Message%2A>| En system-angitt lokalisert beskrivelse. |   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| An empty <xref:System.Guid>.|</xref:System.Guid></xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A></xref:System.Exception.Message%2A></xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException ();
    parameters: []
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid)
  id: '#ctor(System.Guid)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(Guid)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> klassen ved hjelp av en angitt arbeidsflytforekomst <xref:System.Guid>.</xref:System.Guid>"
  remarks: "`instanceId`er det samme som den <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>-egenskapen til arbeidsflytforekomsten.</xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName> Den <xref:System.Exception.Message%2A>for den nye forekomsten initialiseres til en system-angitt melding som beskriver problemet og tar hensyn til gjeldende system kultur.</xref:System.Exception.Message%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| En nullreferanse (`Nothing` i Visual Basic) |   | <xref:System.Exception.Message%2A>| En system-angitt lokalisert beskrivelse. |   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| Den <xref:System.Guid>til arbeidsflytforekomsten, angitt av `instanceId`, for dette unntaket oppstod. |</xref:System.Guid> </xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (Guid instanceId);
    parameters:
    - id: instanceId
      type: System.Guid
      description: "Den <xref:System.Guid>til arbeidsflytforekomsten som dette unntaket oppstod.</xref:System.Guid>"
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> klassen ved hjelp av en angitt feilmelding."
  remarks: "Innholdet i den `message` parameteren er ment å forstå av mennesker. Oppkalleren for denne konstruktøren kreves for å sikre at denne strengen er lokalisert for den gjeldende kulturen i systemet. Hvis du bruker denne konstruktøren må du eksplisitt angi <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>før du throw unntaket.</xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| En nullreferanse (`Nothing` i Visual Basic). |   | <xref:System.Exception.Message%2A>| Feil melding strengen. |   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| An empty <xref:System.Guid>.|</xref:System.Guid></xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A></xref:System.Exception.Message%2A></xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Meldingen som beskriver problemet."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String)
  id: '#ctor(System.Guid,System.String)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(Guid,String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> klassen ved hjelp av en arbeidsflytforekomst som er angitt <xref:System.Guid>og et angitt feilmelding.</xref:System.Guid>"
  remarks: "`instanceId`er det samme som den <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>-egenskapen til arbeidsflytforekomsten.</xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName> Innholdet i den `message` parameteren er ment å forstå av mennesker. Oppkalleren for denne konstruktøren kreves for å sikre at denne strengen er lokalisert for den gjeldende kulturen i systemet.       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| En nullreferanse (`Nothing` i Visual Basic) |   | <xref:System.Exception.Message%2A>| Feil melding strengen. |   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| Den <xref:System.Guid>til arbeidsflytforekomsten, angitt av `instanceId`, for dette unntaket oppstod. |</xref:System.Guid> </xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (Guid instanceId, string message);
    parameters:
    - id: instanceId
      type: System.Guid
      description: "Den <xref:System.Guid>til arbeidsflytforekomsten som dette unntaket oppstod.</xref:System.Guid>"
    - id: message
      type: System.String
      description: "Meldingen som beskriver problemet."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av klassen WorkflowOwnershipException med serialiserte dataene."
  remarks: "Denne konstruktøren kalles under serialisering til reconstitute unntak objektet overføres via en dataflyt.       Se XML og SOAP-serialisering for mer informasjon."
  syntax:
    content: protected WorkflowOwnershipException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "En <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> som inneholder den serialiserte objektdata om som oppstod et unntak."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "En <xref:System.Runtime.Serialization.StreamingContext>som inneholder kontekstavhengig informasjon om kilden eller målet.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> klassen ved hjelp av en angitt feilmelding og en referanse til det interne unntaket er årsaken til dette unntaket."
  remarks: "Et unntak som er registrert som en direkte resultatet av et tidligere unntak må inkludere en referanse til det tidligere unntaket i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>egenskapen returnerer den samme verdien som er oppgitt til konstruktøren, eller en nullreferanse (ingenting i Visual Basic) Hvis den `innerException` parameteren ikke angi verdien for det indre unntaket til konstruktøren.</xref:System.Exception.InnerException%2A> Innholdet i den `message` parameteren er ment å forstå av mennesker. Oppkalleren for denne konstruktøren kreves for å sikre at denne strengen er lokalisert for den gjeldende kulturen i systemet. Hvis du bruker denne konstruktøren må du eksplisitt angi <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>før du throw unntaket.</xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referansen for det indre unntaket. |   | <xref:System.Exception.Message%2A>| Feil melding strengen. |   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| An empty <xref:System.Guid>.|</xref:System.Guid></xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A></xref:System.Exception.Message%2A></xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Meldingen som beskriver problemet."
    - id: innerException
      type: System.Exception
      description: "Unntaket er årsaken til den gjeldende unntaket. Hvis den `innerException` -parameteren er ikke en nullreferanse (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> i Visual Basic), gjeldende unntaket blir opphøyd i en catch-blokk som behandler det interne unntaket."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String,System.Exception)
  id: '#ctor(System.Guid,System.String,System.Exception)'
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: WorkflowOwnershipException(Guid,String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Workflow.Runtime.WorkflowOwnershipException&quot;> </xref> klassen ved hjelp av en arbeidsflytforekomst som er angitt <xref:System.Guid>, feilmeldingen angitte og en referanse til det interne unntaket er årsaken til dette unntaket.</xref:System.Guid>"
  remarks: "Et unntak som er registrert som en direkte resultatet av et tidligere unntak må inkludere en referanse til det tidligere unntaket i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>egenskapen returnerer den samme verdien som er oppgitt til konstruktøren, eller en nullreferanse (ingenting i Visual Basic) Hvis den `innerException` parameteren ikke angi verdien for det indre unntaket til konstruktøren.</xref:System.Exception.InnerException%2A>       `instanceId`er det samme som den <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>-egenskapen til arbeidsflytforekomsten.</xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName> Innholdet i den `message` parameteren er ment å forstå av mennesker. Oppkalleren for denne konstruktøren kreves for å sikre at denne strengen er lokalisert for den gjeldende kulturen i systemet.       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Workflow.Runtime.WorkflowOwnershipException>.</xref:System.Workflow.Runtime.WorkflowOwnershipException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referansen for det indre unntaket. |   | <xref:System.Exception.Message%2A>| Feil melding strengen. |   | <xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A>| Den <xref:System.Guid>til arbeidsflytforekomsten, angitt av `instanceId`, for dette unntaket oppstod. |</xref:System.Guid> </xref:System.Workflow.Runtime.WorkflowOwnershipException.InstanceId%2A> </xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public WorkflowOwnershipException (Guid instanceId, string message, Exception innerException);
    parameters:
    - id: instanceId
      type: System.Guid
      description: "Den <xref:System.Guid>til arbeidsflytforekomsten som dette unntaket oppstod.</xref:System.Guid>"
    - id: message
      type: System.String
      description: "Meldingen som beskriver problemet."
    - id: innerException
      type: System.Exception
      description: "Unntaket er årsaken til den gjeldende unntaket. Hvis den `innerException` -parameteren er ikke en nullreferanse (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> i Visual Basic), gjeldende unntaket blir opphøyd i en catch-blokk som behandler det interne unntaket."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
  type: Method
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Angir den <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> objekt med den <xref:System.Guid>til arbeidsflytforekomsten som er forbundet med denne unntak, og ekstra unntaksinformasjon.</xref:System.Guid>"
  remarks: "GetObjectData angir en <xref:System.Runtime.Serialization.SerializationInfo>med alle objektet unntaksdataene mål for serialisering.</xref:System.Runtime.Serialization.SerializationInfo> Under deserialisering reconstituted unntak-objektet fra den <xref:System.Runtime.Serialization.SerializationInfo>overføres via dataflyten.</xref:System.Runtime.Serialization.SerializationInfo>       Se XML og SOAP-serialisering for mer informasjon."
  syntax:
    content: public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "En <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> som inneholder den serialiserte objektdata om som oppstod et unntak."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "En <xref:System.Runtime.Serialization.StreamingContext>som inneholder kontekstavhengig informasjon om kilden eller målet.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData*
  exceptions:
  - type: System.ArgumentNullException
    commentId: T:System.ArgumentNullException
    description: "<code>info</code>er et nullreferanse (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> i Visual Basic)"
  platform:
  - net462
- uid: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  id: InstanceId
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  langs:
  - csharp
  name: InstanceId
  nameWithType: WorkflowOwnershipException.InstanceId
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  type: Property
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Henter den <xref:System.Guid>til arbeidsflytforekomsten dette unntaket oppstod.</xref:System.Guid>"
  syntax:
    content: public Guid InstanceId { get; set; }
    return:
      type: System.Guid
      description: "InstanceId tilsvarer den &lt;xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName&gt; -egenskapen til arbeidsflytforekomsten."
  overload: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.ArgumentNullException
  isExternal: true
  name: System.ArgumentNullException
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException()
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException()
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException()
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(Guid)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid)
- uid: System.Guid
  parent: System
  isExternal: true
  name: Guid
  nameWithType: Guid
  fullName: System.Guid
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(Guid,String)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.String,System.Exception)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(String,Exception)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor(System.Guid,System.String,System.Exception)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException(Guid,String,Exception)
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.WorkflowOwnershipException(Guid,String,Exception)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: GetObjectData(SerializationInfo,StreamingContext)
  nameWithType: WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData(SerializationInfo,StreamingContext)
- uid: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: InstanceId
  nameWithType: WorkflowOwnershipException.InstanceId
  fullName: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId
- uid: System.Workflow.Runtime.WorkflowOwnershipException.#ctor*
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: WorkflowOwnershipException
  nameWithType: WorkflowOwnershipException.WorkflowOwnershipException
- uid: System.Workflow.Runtime.WorkflowOwnershipException.GetObjectData*
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: GetObjectData
  nameWithType: WorkflowOwnershipException.GetObjectData
- uid: System.Workflow.Runtime.WorkflowOwnershipException.InstanceId*
  parent: System.Workflow.Runtime.WorkflowOwnershipException
  isExternal: false
  name: InstanceId
  nameWithType: WorkflowOwnershipException.InstanceId
